crates/libtiny_tui/src/config.rs

```diff
@@ -26,6 +27,13 @@ pub struct Style {
     pub bg: u16,
 }
 
+#[derive(Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub(crate) enum UiStyle {
```

Comment:
Should we rename this to `Layout` maybe? (both the type name and the config field name)